buildtools/shrinkwrap/shrinkwrap.js

```diff
@@ -22,27 +22,28 @@ var _ = require('lodash'),
 
 module.exports = function (grunt) {
 
-  function removeResolved(json) {
+  function removeResolvedAndFrom(json) {
```

Comment:
fyi @Ujjwalreddy-okta looks like extraneous comes from the dedupe step in install-with-shrinkwrap; created this jira to think about it more: https://oktainc.atlassian.net/browse/OKTA-94191